openmeter/productcatalog/subscription/service/change.go

```diff
@@ -0,0 +1,60 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openmeterio/openmeter/openmeter/productcatalog"
+	plansubscription "github.com/openmeterio/openmeter/openmeter/productcatalog/subscription"
+	"github.com/openmeterio/openmeter/openmeter/subscription"
+	"github.com/openmeterio/openmeter/pkg/models"
+)
+
+func (s *service) Change(ctx context.Context, request plansubscription.ChangeSubscriptionRequest) (plansubscription.SubscriptionChangeResponse, error) {
+	var def plansubscription.SubscriptionChangeResponse
+	var plan subscription.Plan
+
+	if err := request.PlanInput.Validate(); err != nil {
+		return def, &models.GenericUserError{Message: err.Error()}
+	}
+
+	// TODO: let's validate the plan can be changed to!
```

Comment:
pls create ticket and add as `// TODO (OM-XX)`

Reply:
thanks, was left there by mistake, removed it